app/src/main/java/com/it3048/mobiledeviceproject/MainActivity.kt

```diff
@@ -1,41 +1,35 @@
 package com.it3048.mobiledeviceproject
 
-import android.content.DialogInterface
 import android.os.Bundle
-import android.view.ContextThemeWrapper
 import android.view.LayoutInflater
 import android.widget.*
 import android.view.Menu
 import android.view.MenuItem
-import android.widget.EditText
-import android.widget.Spinner
-import android.widget.LinearLayout
-import android.widget.Toast
-import androidx.appcompat.app.AlertDialog
 import androidx.appcompat.app.AppCompatActivity
 import androidx.recyclerview.widget.RecyclerView
 import com.google.android.material.floatingactionbutton.FloatingActionButton
-import com.google.android.material.snackbar.Snackbar
-import kotlinx.android.synthetic.main.activity_main.*
-import java.util.*
 import kotlin.collections.ArrayList
 
 
 class MainActivity : AppCompatActivity() {
 
     private var addMeetingForm = AddMeetingForm()
-    lateinit var recyclerView: RecyclerView
+
+    private lateinit var recyclerView: RecyclerView
```

Comment:
Changed visibility modifier according to coding conventions.
https://kotlinlang.org/docs/coding-conventions.html